### PR TITLE
Added sorting option in features

### LINKED ISSUE #7485: Sorting in features page
**Please describe the desired behavior.**
Describe the solution you'd like.
Currently there is no kind of sorting on features page .It would be benficial if we could sort features based on title or last edited.

**Additional context (optional)**
Feel free to include any applicable: code, screenshots, or gifs.

### PR DESCRIPTION
**Pull request by Manasa2850**
Fixes #7485 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by Manasa2850** [comment]
ezgif com-video-to-gif

File: app/controllers/features_controller.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class FeaturesController < ApplicationController
   def index
     @title = "Features"
     @features = Node.where(type: 'feature')
-      .paginate(page: params[:page])
+    @features = Node.sort_features(@features, params[:sort]).paginate(page: params[:page])
```
Comment (cesswairimu) [inline app/controllers/features_controller.rb:7]: Hi, @Manasa2850 , maybe we should rename one of these variables, what do you think?

Everything else looks great thanks  a bunch for working on this

Comment (Manasa2850) [reply app/controllers/features_controller.rb:7]: Hi @cesswairimu, sure I'll change that. Thank you!

**Review by jywarren — APPROVED** [review]
Looks really nice, thanks!!!

**Pull request merged**